tool/flutter_site/lib/src/commands/check_links.dart

```diff
@@ -96,7 +97,10 @@ Future<int> _checkLinks({bool checkExternal = false}) async {
     }
   } finally {
     print('Shutting down Firebase hosting emulator...');
-    emulatorProcess.kill();
+    // Terminate the process, first sending two sigterm signals for
+    // a chance to gracefully handle the signal.
+    emulatorProcess.kill(ProcessSignal.sigterm);
+    emulatorProcess.kill(ProcessSignal.sigterm);
```

Comment:
There should probably be a sleep here to give the receiving process time to shut down.

Reply:
Ah good point, thanks. Done!